query/query_test.go

```diff
@@ -89,27 +90,36 @@ var parseOnly = []string{
 
 // these queries should fail with a syntax error.
 var syntaxErrorQuery = []string{
-	"select (",
-	"select )",
-	"describe (",
-	"describe invalid_regex where key matches 'ab['",
-	"select x invalid_property 0",
-	"select x sampleby 0",
-	"select x sample 0",
-	"select x by 0",
+	"select ( from 0 to 0",
+	"select ) from 0 to 0",
+	"describe ( from 0 to 0",
+	"describe invalid_regex where key matches 'ab[' from 0 to 0",
+	"select x invalid_property 0 from 0 to 0",
+	"select x sampleby 0 from 0 to 0",
+	"select x sample 0 from 0 to 0",
+	"select x by 0 from 0 to 0",
+	"select x",
+	"select x from 0",
+	"select x to 0",
+	"select x from 0 from 1 to 0",
+	"select x from 0 to 1 to 0",
+	"select x from 0 resolution 30 resolution 25 to 0",
+	"select x from 0 from 1 sample by 'min' sample by 'min' to 0",
 }
 
 func TestParse_success(t *testing.T) {
 	for _, row := range inputs {
-		if err := checkSyntaxError(t, row); err != nil {
+		_, err := Parse(row)
+		if err != nil {
 			t.Errorf("[%s] failed to parse: %s", row, err.Error())
 		}
 	}
-	for _, row := range parseOnly {
-		if err := checkSyntaxError(t, row); err != nil {
+	/*for _, row := range parseOnly {
```

Comment:
Originally it broke and made things hard to debug - now it's fine so I put it back